README.md

```diff
@@ -18,39 +18,121 @@ based on the Open Cluster Management project.
+1. Import the near edge cluster(s).
+   In the OpenShift Console this can be done in All Clusters menu > Infrastructure > Clusters > Cluster list tab.
+   In the Cluster set popup menu, select the cluster set you created for the near edge clusters.
+   If you use the Import mode of "Enter your server URL and API token for the existing cluster",
+   obtain the address of the server and the token on the other cluster's OpenShift Console 
+   at the top-right corner user menu > Copy login command, for an admin user.
+
+#### Create namespaces for applications
+
+For each of the deployed models-turned-into-application, we will create separate namespace/project on the ACM hub cluster.
+The ACM setup in this repository expects namespaces `tensorflow-housing-app` and `bike-rental-app`.
+
+As a regular user in the OpenShift Console, in the local-cluster > Developer menu, Project popup, use the "Create Project" button;
+alternatively, run
+```bash
+oc new-project tensorflow-housing-app
+oc new-project bike-rental-app
+```
+
+#### Bind namespaces to the cluster set
+
+The newly created namespaces need to be bound to the cluster set that contains the near edge clusters.
+
+On the ACM hub cluster, in the OpenShift Console in the All Clusters menu > Infrastructure > Clusters > Cluster sets tab,
+click the three-dot button at the right of the cluster set, select Edit namespace bindings, and add the namespaces you've created.
+
+This is the last step that needs to be done as an admin user with `cluster-admins` privileges.
+
+#### Deploy the applications to remote clusters
+
+With the remote near edge clusters now in the cluster set and that cluster set bound to the application namespaces,
+it is now possible to deploy the applications to remote clusters through
+ACM [Channels](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html-single/applications/index#channels),
+[Subscriptions](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html-single/applications/index#subscriptions),
+[Placements](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html-single/applications/index#placement-rules),
+and [Applications](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html-single/applications/index#applications).
+
+We will create the objects in the respective namespaces from the `acm/registration/` directory structure.
+
+However, it is important to note that the actual application configuration, stored in `acm/odh-edge/`,
+does not come from our local git repository checkout.
+Instead, [`acm/registration/near-edge/overlays/tensorflow-housing-app/kustomization.yaml`](acm/registration/near-edge/overlays/tensorflow-housing-app/kustomization.yamlacm/registration/near-edge/overlays/tensorflow-housing-app/kustomization.yaml)
```

Comment:
```suggestion
Instead, [`acm/registration/near-edge/overlays/tensorflow-housing-app/kustomization.yaml`](acm/registration/near-edge/overlays/tensorflow-housing-app/kustomization.yaml)
```

Reply:
Ouch. Applied in 171844e691d75ec81b322b98817751b7dd7097a6.